goal_pub_topic: "/planning/click_goal"
odom_topic: "/lidar_slam/odom"
# odom_topic: "/mavros/local_position/odom"
# 0 for RVIZ and; 1 for Mavros; 2 for constant time
start_trigger_type: 2
# trigger after start_trigger_delay (s)
start_program_delay: 3.0
# 0 for PATH and 1 for waypoints
cmd_type: 1
# the distance threshold for waypoint goal swithcing
switch_dis: 3.0
# odom timeout threshold (s)
odom_timeout: 0.1
# goal publish dt (s), set to inf if the goal only need to pub once
publish_dt: 99999.0
# mission waypoints, must be in the same dimension
#    waypoints_file_name: "a_working_waypoints.txt"
#waypoints_file_name: "benchmark_dense.txt"
